pybuildkite/jobs.py

```diff
@@ -61,3 +61,48 @@ def get_job_environment_variables(self, organization, pipeline, build, job):
         return self.client.get(
             self.path.format(organization, pipeline, build, job) + "env"
         )
+
+    def retry_job(self, organization, pipeline, build, job):
+        """
+        Retries a failed or timed_out job.
+         :param organization: Organization slug
+        :param pipeline: Pipeline slug
+        :param build: Build number
+        :param job: Job id
+        :return: response
+        """
+        return self.client.put(
+            self.path.format(organization, pipeline, build, job)+"retry"
+        )
+
+    def unblock_job(self, organization, pipeline, build, job, fields, unblocker=None):
+        """
+           Unblocks a build’s "Block pipeline" job.
+            :param organization: Organization slug
+           :param pipeline: Pipeline slug
+           :param build: Build number
+           :param job: Job id
+           :fields: name and email
+           :unblocker: The user id of the person activating the job
+           :return: response
+        """
+        body ={
+            "field" : fields,
+            "unblocker" : unblocker
+        }
+        return self.client.put(
+            self.path.format(organization, pipeline, build, job)+"unblock", body=body
```

Comment:
Two things. won't it need to be `/unblock`? And can we follow the same structure here https://github.com/pyasi/pybuildkite/blob/master/pybuildkite/builds.py#L259 just for consistency (minor change).

Thanks!